sdk/postgresql/Azure.ResourceManager.PostgreSql/CHANGELOG.md

```diff
@@ -1,5 +1,29 @@
 # Release History
 
+## 1.2.0 (2023-10-25)
```

Comment:
Need to specify a future date. Next time, please reserve some buffer time for review/modification. Thanks.

Reply:
Updated to November ship date :-)